tekton/pipelines/binary-container.yaml

```diff
@@ -78,7 +78,14 @@ spec:
       runAfter:
         - clone
       taskRef:
-        name: binary-container-init-0-1
+        resolver: git
```

Comment:
this seems like some testing change which should be reverted

Reply:
yes, it's from commit `DO NOT MERGE`. I'll remove it before merging